### ISSUE TITLE
Asking for sudo password twice.

### ISSUE THREAD

**Issue by marc-marcos** [SolDoesTech/HyprV4#95]
When using the script it asks for the password once in the start but then when it's installing swww it takes a disproportionate amount of time (close to 5 minutes) and asks for the root password again while still printing dots.

Tried in bare metal, with fresh archinstall install, MSI, no graphics card, Intel processor.

**Comment by fallenexistence0** [comment]
Did you enter the password incorrectly? Try canceling and restarting the install.

**Comment by marc-marcos** [comment]
Nope, don't think so, the first half of the installation or so work fine, then prompted to enter the password again. When entered the script finished correctly.

**Comment by fallenexistence0** [comment]
That's pretty normal when using linux, I think it has to do with sudo timing out, or just for confirmation. Hope you enjoy your time using linux!
